### PR TITLE
jackett: 0.17.668 -> 0.17.677

### PR DESCRIPTION
**Pull request by andreisergiu98**
I also replaced mono with dotnet core to fix a warning on the jackett webpage.
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/jackett/default.nix [hunk 1]
```diff
@@ -15,11 +15,16 @@ stdenv.mkDerivation rec {
     mkdir -p $out/{bin,share/${pname}-${version}}
     cp -r * $out/share/${pname}-${version}
 
-    makeWrapper "${mono}/bin/mono" $out/bin/Jackett \
-      --add-flags "$out/share/${pname}-${version}/JackettConsole.exe" \
-      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ curl icu60 openssl zlib ]}
+    makeWrapper "${dotnetCorePackages.netcore_3_1}/bin/dotnet" $out/bin/Jackett \
+      --add-flags "$out/share/${pname}-${version}/jackett.dll" \
+      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [
+          curl icu60 openssl zlib ]}
```
Comment (SuperSandro2000) [inline pkgs/servers/jackett/default.nix:21]:
```suggestion
      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ curl icu60 openssl zlib ]}
```

**Comment by andreisergiu98** [comment]
Now I realize that moving to dotnet core means dropping i686 and I don't know if that's desired if it can be avoided. I think we could also make a rule to use the mono build for i686 or we could keep mono for each platform and drop this PR.

**Comment by SuperSandro2000** [comment]
> Now I realize that moving to dotnet core means dropping i686 and I don't know if that's desired if it can be avoided.

You think anyone on i686 would use jackett? The platform is really unpopular now days and I have no problem with dropping it.

**Comment by purcell** [comment]
I smoke-tested this change on Darwin and it's fine there fwiw.

I'd note that I was the one who switched this derivation _from_ platform-specific binaries to the mono version, largely because a few similar derivations do that too (sonarr, radarr, jellyfin IIRC), and it's somewhat less maintenance overall. The trade-off is that there's then a first-run notification in the app suggesting that a platform-specific binary be used instead. I don't know what approach is best going forward, just wanted to share this context.

**Comment by andreisergiu98** [comment]
@purcell I agree so I will close this PR for now. Jackett makes releases almost everyday so having a bot making PRs makes it easier to update.

Maybe in the future it might be worth it to create an `updateScript` similar with [Radarr](https://github.com/NixOS/nixpkgs/pull/115938), but for now I don't think so, mainly because the warning is not that annoying and can be hidden with ublock or other browser addons, also I don't think users are using the web client that much.

**Pull request closed without merging**